tests/prague/test_state_transition.py

```diff
@@ -1,32 +1,20 @@
 from functools import partial
-from typing import Dict, Generator, Tuple
+from typing import Dict
 
 import pytest
 
-from tests.helpers import TEST_FIXTURES
+from tests.helpers import EEST_TESTS_PATH, ETHEREUM_TESTS_PATH
 from tests.helpers.load_state_tests import (
     Load,
     fetch_state_test_files,
     idfn,
     run_blockchain_st_test,
 )
 
-fetch_prague_tests = partial(fetch_state_test_files, network="Prague")
-
-FIXTURES_LOADER = Load("Prague", "prague")
-
-run_prague_blockchain_st_tests = partial(
-    run_blockchain_st_test, load=FIXTURES_LOADER
-)
-
-ETHEREUM_TESTS_PATH = TEST_FIXTURES["ethereum_tests"]["fixture_path"]
-ETHEREUM_SPEC_TESTS_PATH = TEST_FIXTURES["execution_spec_tests"][
```

Comment:
Is `TEST_FIXTURES["execution_spec_tests"]` now dead? If so, you should probably remove it.

Reply:
You can probably also remove `download_fixtures()` from `conftest.py` as dead as well.

Reply:
Updated